source/api/presentation/2.1/index.md

```diff
@@ -320,7 +320,8 @@ A link to a machine readable document that semantically describes the resource w
 ##### within
 A link to a resource that contains the current resource, such as annotation lists within a layer. This also allows linking upwards to collections that allow browsing of the digitized objects available.
 
- * Any resource type _MAY_ be within one or more containing resources.
+ * Collections or annotation lists that serve as [pages][paging] _MUST_ be within exactly one paged resource.
+ * Other resource types, and collections or annotation lists not serving as pages, _MAY_ be within one or more containing resources.
```

Comment:
I'd prefer:

> Other resource types, including collections or annotation lists not serving as pages, _MAY_ be within one or more containing resources.

collections or annotation lists are still resources, "and" implies something else....but it's a tiny thing.